partcad/src/partcad/healthcheck/openscad.py

```
@@ -0,0 +1,232 @@
+
+            result = subprocess.run(["sudo", "apt-get", "update"], check=True, capture_output=True)
+
+            if result.returncode != 0:
+                pc_logging.error("OpenSCAD installation failed.")
+                pc_logging.debug(result.stderr)
+                return False
+
+        except subprocess.CalledProcessError as e:
+            pc_logging.error("Failed to update apt package index.")
+            pc_logging.debug(e)
+        except Exception as e:
```

Ruff (0.11.9)

Local variable `e` is assigned to but never used

Remove assignment to unused variable `e`

(F841)